.github/dependabot.yml

```diff
@@ -23,6 +23,25 @@ updates:
     reviewers:
       -  aws/serverless-application-experience-sbt
     open-pull-requests-limit: 10
+    groups:
+      boto-stubs:
+        patterns:
```

Comment:
can we include boto3 and botocore libraries themselves to this group?

Reply:
Sure, that's something we can include in the boto group.